samples/highcharts/studies/map-activate/demo.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * Snippet to activate the map navigation on click.
+ *
+ * To do
+ * - Option for the button to remain hidden until the user interacts with the
+ *   chart.
+ * - Option to change the map's state back
+ * - Make the button HTML
+ * - Make sample and blog article
+ */
+
```

Comment:
@TorsteinHonsi Should it be here? 👀

Reply:
Sorry, it was not meant to be. But it doesn't hurt, so let's just leave it there. It will soon end up in the master.